### ISSUE TITLE
terminal completions are not indicated to screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#235022]
1. enable `terminal.integrated.suggest.enabled` and (optional) `terminal.integrated.suggest.enableExtensionCompletions`
2. use a screen reader and enter screen reader mode
3. type in the terminal, for example, `c`
4. completions widget is visible, but it's not announced to screen reader users

Completions in the editor aren't announced to screen reader users, but when they're invoked via `ctrl+space`, they are and say the suggestion then x of count.

cc @jooyoungseo

**Comment by meganrogge** [comment]
We'll need to fix this first https://github.com/microsoft/vscode/issues/233921

**Comment by jooyoungseo** [comment]
It's verbally announced on Windows when tested with the latest JAWS. However, the terminal suggestion is not presented in a braille display.

**Comment by Tyriar** [comment]
@meganrogge my guess is this is either the terminal's announcing conflicting with it or the simple suggest widget missing some a11y stuff.
